MetaMask/metamask-extension
# fix: change asset picker button to buttonbase to fix text theming

## **Description**


The asset picker in dark mode previously showed the down icon and token symbol in their light mode variants, instead of dark mode. This is because `Button` defaults to the primary variant which hard codes the theme as light mode for all descendant elements. This PR changes the AssetPicker main component from Button to ButtonBase since it does not default to light mode

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27127?quickstart=1)

## **Related issues**

Fixes: #27046 

## **Manual testing steps**

1. Change to dark mode
2. Open Send Page
3. Asset symbol and down carat should both render as white
4. Switch to light mode
5. Asset symbol and down carat should both render as black

## **Screenshots/Recordings**

### **Before**

<img width="1624" alt="image" src="https://github.com/user-attachments/assets/69e086b1-daec-46d9-b299-74434d58a616">


### **After**

<img width="1624" alt="image" src="https://github.com/user-attachments/assets/709898ae-9766-4902-a089-9183bac8fcc1">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
